src/pages/comparison.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ShopHeader from '../shop_header/shop_header';
+import DiscountShop from '../discount_shop/discount_shop';
+import ComparisonBody from '../comparisonBody/comparisonBody';
+const Comparison = () => {
+    return (
+        <div>
+            <ShopHeader logo={true} path={'Product Comparison'} />
```

Comment:
looking at this code I'd say that `path` prop would expect some path, not a title